controllers/projectController.js

```diff
@@ -1,6 +1,7 @@
 //Import the dependencies
-import { Router } from 'express';
```

Comment:
Since router is now handled in script.js, I removed it from here.
Also, model is not used in the file. That is why removed.